app/javascript/components/Articles/ArticlesDetails.tsx

```diff
@@ -1,25 +1,25 @@
 import React from 'react'; 
 
-export interface ArticleDetailsProps {
+export interface ArticlesDetailsProps {
   title:string,
   content:string,
   user:string,
   created_at:string
 }
  
-const ArticleDetails: React.FC<ArticleDetailsProps> = (props) => {
+const ArticlesDetails: React.FC<ArticlesDetailsProps> = (props) => {
```

Comment:
An observation here is that this react component is very similar to FullArticle. You could possibly roll both into a generic DisplayArticle (or something like that) react component?

Reply:
And with different header tags this could be useful https://stackoverflow.com/questions/33471880/dynamic-tag-name-in-jsx-and-react. That or you could abstract the parts that are similar to another component.